interfaces/cython/cantera/thermo.pxd

```diff
@@ -199,12 +199,14 @@ cdef extern from "cantera/thermo/PlasmaPhase.h":
         string electronEnergyDistributionType()
         void setQuadratureMethod(const string&) except +translate_exception
         string quadratureMethod()
-        void enableNormalizeElectronEnergyDist(cbool) except +translate_exception
+        void enableNormalizeElectronEnergyDist(cbool)
         cbool normalizeElectronEnergyDistEnabled()
         void setMeanElectronEnergy(double) except +translate_exception
         double isotropicShapeFactor()
         double meanElectronEnergy()
         size_t nElectronEnergyLevels() except +translate_exception
+        double enthalpy_mole() except +translate_exception
```

Comment:
You shouldn't need to declare this, since it's already a member of the base `ThermoPhase` class. Although, it looks like the Cython declaration here doesn't indicate that `PlasmaPhase` is derived from `ThermoPhase`.

Reply:
Can Cython detect that PlasmaPhase is derived from ThernoPhase (C++)?